<?xml version="1.0"?>
<launch>	
	<include file="$(find rrt_exploration)/launch/single.launch" />
	<!--<include file="$(find rrt_exploration_tutorials)/launch/single_simulated_house.launch" />-->
	
	<include file="$(find exp_devel)/launch/kobuki_box_world.launch" >
		<arg name="gmap_topic" value="scan_fix"/>
		<arg name="gmap_maxUrange" value="5.5"/>
		<arg name="gmap_maxRange" value="6.1"/>
	</include>
	
	<node name="laser_hot_fix" pkg="exp_devel" type="laser_hot_fix"  output="screen">
		<param name="max_range" value="6.0" />
		<param name="sub_scan_topic" value="robot_1/base_scan" />
		<param name="pub_scan_topic" value="robot_1/scan_fix" />
	</node>
	
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find rrt_exploration_tutorials)/launch/includes/rviz_config/single.rviz">
		<remap from="move_base_simple/goal" to="robot_1/move_base_simple/goal"/>
	</node>

</launch>